### DISCUSSION TITLE
Srs showing as @localdomain instead of @domain

### DISCUSSION THREAD

**Discussion by symeonmanis** [roehling/postsrsd#121] [Q&A] [answered]
Hello all,

I have added those lines in my main.cf

```
# PostSRSd settings.
sender_canonical_maps = tcp:127.0.0.1:10001
sender_canonical_classes = envelope_sender
recipient_canonical_maps = tcp:127.0.0.1:10002
recipient_canonical_classes= envelope_recipient,header_recipient
```

Then in Gmail I got SPF neutral 

```
ARC-Authentication-Results: i=1; mx.google.com;
       spf=neutral (google.com: 168.119.237.152 is neither permitted nor denied by best guess record for domain of srs0=n0ge=r4=mydomain.gr=simos@localdomain)

```
Any ideas what is wrong?

**Comment by roehling** [comment, marked as answer]
PostSRSd uses the Postfix config value [mydomain](http://www.postfix.org/BASIC_CONFIGURATION_README.html#mydomain) by default. You should probably fix your Postfix configuration. You can also configure the domain in `/etc/default/postsrsd` if you want to rewrite to a different domain than the one used by Postfix.

**Reply by symeonmanis** [reply]
Thanks a lot 👍. I configured the domain in `/etc/default/postsrsd` and now it works.
